app.js

```diff
@@ -11,6 +11,9 @@ var webpayPlusMallRouter = require("./routes/webpay_plus_mall");
 var webpayPlusMallDeferredRouter = require("./routes/webpay_plus_mall_deferred");
 
 var app = express();
+if (app.settings.env == "development") {
+  require("dotenv").config();
+}
```

Comment:
Why you only require .env on development?

Reply:
Because in heroku is easier to load env vars without dot env